src/de.rs

```diff
@@ -424,3 +434,10 @@ pub fn from_reader_xml<R: Read, T: de::DeserializeOwned>(reader: R) -> Result<T,
     let mut de = Deserializer::new(reader);
     de::Deserialize::deserialize(&mut de)
 }
+
+/// Interprets a [`plist::Value`] as an instance of type `T`.
```

Comment:
`plist::Value` doesn't work with rustdoc here, because `plist` isn't in scope here. You can either use `Value` directly, or write `crate::Value`, or if you want the link to read `plist::Value`, you can do ``[`plist::Value`](Value)``